src/rom_header_rhh.c

```diff
@@ -21,6 +22,9 @@ struct RHHRomHeader
     /*0x0C*/ u16 numSpecies;
     /*0x0E*/ u16 abilitiesCount;
     /*0x10*/ const struct Ability *abilities;
+    /*0x14*/ u16 itemsCount;
+    /*0x16*/ u8 itemNameLength;
+    /*0x17*/ u8 padding[3];
```

Comment:
Now that `itemsCount` is u16, there's no need for the 2 extra bytes of padding.
```suggestion
    /*0x17*/ u8 padding;
```